pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -83,6 +85,62 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  selected: boolean;
+}
+
+function TabPanel(props: TabPanelProps) {
+  return (
+      <div
+          role="tabpanel"
+          hidden={!props.selected}
+          id={`simple-tabpanel-${props.index}`}
+          aria-labelledby={`simple-tab-${props.index}`}
+      >
+        {props.selected && (
+            <Box >
```

Comment:
```suggestion
            <Box>
```
nits, and other `<Box >` too